content/en/blog/2022/why-and-how-ebay-pivoted-to-opentelemetry/index.md

```diff
@@ -0,0 +1,348 @@
+---
+title: "Why and How eBay Pivoted to OpenTelemetry"
+linkTitle: eBay OpenTelemetry
+date: 2022-12-13
+author: "[Vijay Samuel](https://github.com/vjsamuel) (eBay)"
+canonical_url: https://tech.ebayinc.com/engineering/why-and-how-ebay-pivoted-to-opentelemetry/
+---
+
+_A version of this article was [originally posted][] on the eBay Tech Blog._
```

Comment:
For the last cross-posted blog posts we put this towards the end of the page.

Reply:
done.